### PR TITLE
Fixed #632: '9-dot navigation menu to other apps not closing'

### LINKED ISSUE #632: 9-dot navigation menu to other apps not closing
**I'm submitting a ...**
- [ ] bug report
- [X] feature request

**Current behavior:**

The 9-dot menu (navigation to fossasia apps) at top-right side is not closing when the mouse is clicked anywhere else on the page, except at the 9-dot icon itself.

**Expected behavior:**

The opened menu should automatically get close when the mouse is clicked anywhere on the page.

**Screenshots of the issue:**

screenshot from 2018-01-04 18-20-18
screenshot from 2018-01-04 18-20-37

### PR DESCRIPTION
**Pull request by simsausaurabh**
**Changes proposed in this pull request**
- Made changes in service-box component to close navigation menu on top right side when clicked outside

**Link to live demo: ** http://pr-652-fossasia-loklaksearch.surge.sh

**Closes #632 #638**

### CODE REVIEW

File: src/app/service-box/service-box.component.scss [hunk 1]
```diff
@@ -2,13 +2,21 @@
...
+}
+
 .box {
 	border: 1px solid #ccc;
 	box-shadow: 0 2px 10px rgba(0,0,0,.2);
 	padding: 30px;
 	position: absolute;
-	right: 0;
-	top: 45px;
+    right: 0px;
```
Comment (singhpratyush) [inline src/app/service-box/service-box.component.scss]: Looks like some indentation issue. Please fix.

File: src/app/service-box/service-box.component.html [hunk 2]
```diff
@@ -90,6 +88,7 @@
 					<span>Eventyay</span>
 				</div>
 			</a>
+			</div>
 		</div>
 	</div>
-</div>
+</div>
```
Comment (singhpratyush) [inline src/app/service-box/service-box.component.html]: Please introduce a newline after this.

**Comment by simsausaurabh** [comment]
@singhpratyush please review.

File: src/app/service-box/service-box.component.scss [hunk 3]
```diff
@@ -2,13 +2,21 @@
...
+}
+
 .box {
 	border: 1px solid #ccc;
 	box-shadow: 0 2px 10px rgba(0,0,0,.2);
 	padding: 30px;
 	position: absolute;
-	right: 0;
-	top: 45px;
+	right: 0px;
```
Comment (singhpratyush) [inline src/app/service-box/service-box.component.scss]: Also, this change shouldn't be there. We should omit the unit with zero values in css.

Comment (simsausaurabh) [reply src/app/service-box/service-box.component.scss]: Are you referring to remove 'right' parameter from css which I have included ?

Comment (singhpratyush) [reply src/app/service-box/service-box.component.scss]: No, just remove the unit.

Comment (simsausaurabh) [reply src/app/service-box/service-box.component.scss]: Okay, thanks. I'm making the changes.

Comment (simsausaurabh) [reply src/app/service-box/service-box.component.scss]: @singhpratyush there are some other parameters in the given scss file with 0 value and px unit shall I remove them also?

**Comment by simsausaurabh** [comment]
@singhpratyush please review.

**Pull request merged**